### ISSUE TITLE
Parallel version

### ISSUE THREAD

**Issue by mariavd** [qmcurrents/gimic#176]
This is not an issue but a question. I compiled with the ``--omp`` flag and submitted a calculation of the current density in 3D on 8 cores, specifying

```
export OMP_NUM_THREADS=8
export OMP_STACKSIZE=16M
```

Is it supposed to print that it indeed is running on 8 cores? The job is rather big, and ETA does not fit into the format specifier, so I won't be able to tell if it indeed ran faster than serial.

**Comment by heikef** [comment]
You can also add --blas to speed it up when compiling.

**Comment by heikef** [comment]
https://github.com/qmcurrents/gimic/issues/170

**Comment by bast** [comment]
Right, the timing is not correct in case of OMP. As to BLAS, it is very important to use optimized blas, most probably MKL but for this we need to fix GIMIC issues on Intel.

**Comment by mariavd** [comment]
Ok, thanks. Possibly it might be useful to print an extra line with the number of threads in use but this is of course just a suggestion.

**Comment by mariavd** [comment]
Just to let you know that I added a line to print the ETA in hours in my fork. It looks like this:
```
 INFO: Estimated CPU time for single core calculation:  4781999.10 sec (1328.3 h )
```
I could make a pull request in case you find it useful. You don't have to, of course, since it's barely significant.

**Comment by bast** [comment]
I think this is useful.

**Comment by mariavd** [comment]
This issue should have been closed by #178. My fault that I didn't mark it.
